app/models/timeline/monitoring_event.rb

```diff
@@ -0,0 +1,28 @@
+class Timeline::MonitoringEvent < Timeline::Event
+  include ActionView::Helpers::TagHelper
+
+  attr_reader :review
+  delegate :api_error_message, :status, :review_type, :outcome,
+    to: :review
+
+  def initialize(issue_monitoring_review)
+    @name = issue_monitoring_review.review_id
+    @date = issue_monitoring_review.start_date
+    @review = issue_monitoring_review.monitoring_review
+  end
+
+  def api_call_succeeded?
+    review.valid?
+  end
+
+  def review_link
+    # using content_tag for the link will properly escape any dangerous characters that sneak into name or monitoring url
+    content_tag(:a, name, class: "usa-link", href: review.itams_url, target: "_blank")
+  rescue Api::Error
```

Comment:
[question] where does this get raised?

Reply:
Good catch, I had copied that over from the TTA activity report without thinking about how I pulled out the `get_api_data_field` error for monitoring. If you think just commenting it for now is the way to go, I can do that and I'll add a note on the error issue.

Reply:
Yeah, I think it's fine as long as the person implementing the error task is super aware of this being here